spk/beets/Makefile

```diff
@@ -1,17 +1,19 @@
 SPK_NAME = beets
 SPK_VERS = 1.4.9
-SPK_REV = 4
+SPK_REV = 5
 SPK_ICON = src/beets.png
 
-PIP = pip3
```

Comment:
It turns out that the version used by the spk was built using the environment’s python 3 installation.